client/src/components/Login/index.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Guest from '@/assets/images/guest_icon.png';
+import { GithubIcon } from '@/assets/svg/svgIcons';
+
+export interface ILoginProps {
+  onGuestLoginClick: () => void;
+  onGithubLoginClick: () => void;
```

Comment:
```suggestion
  onGuestLogin: () => void;
  onGithubLogin: () => void;
```

이 친구들도 위의 리뷰와 같은 맥락으로,
어느 이벤트인지까지 상위 컴포넌트에서 알 필요 없을 것 같아요 ~ ☺️

Reply:
그런 방법이 있꾼요~~ 좋습니다!